plasmapy/physics/transport.py

```diff
@@ -201,7 +205,1511 @@ def Coulomb_logarithm(T, n_e, particles, V=None):
     # transport theory, we shall celebrate by returning the Coulomb
     # logarithm.
 
-    ln_Lambda = np.log(b_max/b_min)
+    ln_Lambda = np.log(b_max / b_min)
     ln_Lambda = ln_Lambda.to(units.dimensionless_unscaled).value
 
     return ln_Lambda
+
+
+class classical_transport:
+    r"""
+    Braginskii-esque classical transport coefficients
+
+    Notes
+    -----
+    Classical transport theory is derived by using kinetic theory to close the
+    plasma two-fluid (electron and ion fluid) equations in the collisional
+    limit. The first complete model in this form was done by S. I. Braginskii.
+
+    This function uses fitting functions from literature to calculate
+    the transport coefficients, which are the resistivity, thermoelectric
+    conductivity, thermal conductivity, and viscosity, which can be used
+    to close the two-fluid equations.
+
+    Note well the assumptions in the derivation of classical transport.
+    It is assumed that turbulent transport does not dominate, and the velocity
+    distribution function is close to Maxwellian (no extremely strong
+    gradients), which is equivalent to the following conditions:
+
+    * collisional frequency >> gyrofrequency
+    * collisional mean free path << gradient scale length along field
+    * gyroradius << gradient scale length perpendicular to field
+
+    When classical transport is not valid, e.g. due to the presence of strong
+    gradients or turbulent transport, the transport is significantly increased
+    by these other effects. Thus, classical transport often serves as a lower
+    bound on the losses / transport encountered in a plasma.
+
+    Models implemented:
+
+    Braginskii [1]_
+    The original Braginskii treatment as presented in the highly cited review
+    paper from 1965. Coefficients are found from expansion of the kinetic
+    equation in Laguerre / Sonine polynomials, truncated at K = 2. This theory
+    allow for arbitrary Hall parameter and include results for Z = 1, 2, 3, 4,
+    and infinity (Lorentz gas / stationary ion approximation.)
+
+    Spitzer-Harm [2]_ [3]_
+    These coefficients were obtained from a numerical solution of the Fokker-
+    Planck equation. They give one of the earliest and most accurate (in the
+    Fokker-Planck sense) results for electron transport in simple plasma. They
+    principally apply in the unmagnetized / parallel field case, although for
+    resistivity Spitzer also calculated a famous result for a strong
+    perpendicular magnetic field. Results are for Z = 1, 2, 4, 16,
+    and infinity (Lorentz gas / stationary ion approximation.)
```

Comment:
Minor punctuation point:  In this and previous paragraph, move the final period outside of the parentheses.

Reply:
Done.